### DISCUSSION TITLE
nginx process is failing to start

### DISCUSSION THREAD

**Discussion by dev30k** [outline/outline#6114] [Self Hosting]
Nginx failing to start after hosting the repo on AWS using the steps given on oultine-onprem

**To Reproduce**
1. Use the steps described on outline onprem for cloud hosting on AWS

**Expected behavior**
The program should run after deploying to AWS but the Nginx if failing to start.

**Screenshots**
https://i.stack.imgur.com/7tQzI.png

**Outline (please complete the following information):**
- Install: self hosted
- Version: latest

**Desktop (please complete the following information):**
 - OS: AWS Ubuntu
 - Browser Firefox
 - Version Latest

**Comment by tommoor** [comment]
Given the errors is that the upstream host cannot be found – what are the logs associated with the `outline` container?

**Reply by dev30k** [reply]
Yeah sure here are the logs but let me explain what i did first:

1. Here is my docker compose file 
![image](https://github.com/outline/outline/assets/66430325/0b142772-1fc5-4ed5-8ae0-2133f19ab4ee)

2. I edited the docker.env file and included all the required urls for the database to communicate with the application. Here is my output when i run database.setup.sh file
![image](https://github.com/outline/outline/assets/66430325/5ec961cc-bdc1-4a99-baeb-cdb27c12ab48)
3. Here a brief of my docker.env file
![image](https://github.com/outline/outline/assets/66430325/65de25c1-16d0-4792-a96e-a95638d4bc25)
4. Here is the output on my docker logs
![image](https://github.com/outline/outline/assets/66430325/0a5e81bc-eda9-47cb-aa7f-ee35692e1146)

NOTE: I have disabled rds.force_ssl parameter group in my AWS RDS. The amazon RDS  is not publicly  accessible they communicate with EC2 via a security group. For more clarification here is how i setup my EC2 and RDS : https://aws.amazon.com/getting-started/hands-on/deploy-drupal-with-amazon-rds/1/

**Reply by tommoor** [reply]
The enterprise image will not run without a license key, are you sure it isn't shutting down due to that? Try the community edition image `outlinewiki/outline`

**Reply by dev30k** [reply]
I tried you suggestion and here is the output but it still not working although i have seen some progress
![image](https://github.com/outline/outline/assets/66430325/0b367b14-a008-4380-9352-03e90945bf53)
![image](https://github.com/outline/outline/assets/66430325/f22d8fe3-4ddd-4e45-82e3-e87c0c373d92)

For your reference here is my security group
![image](https://github.com/outline/outline/assets/66430325/8146248b-4c19-4bc0-b3ae-2f80b4615572)

I have also disabled firewall in  my EC2

**Reply by dev30k** [reply]
Waiting for your response

**Reply by tommoor** [reply]
In terms of AWS security groups etc I have no idea, you're still only posting snippets of the logs and none of the logs from the outline container so it's hard to say more.

**Reply by dev30k** [reply]
Might i suggest a short video call of 30 mins whenever you are free

**Reply by dev30k** [reply]
The logs i posted is from the docker container from the command sudo docker-compose logs -f

**Reply by tommoor** [reply]
You need to post the full logs from server startup for a chance at debugging this issue.

**Reply by dev30k** [reply]
Here it is:
![image](https://github.com/outline/outline/assets/66430325/98f0522b-5436-4c19-b145-1df618894772)
![image](https://github.com/outline/outline/assets/66430325/d2af2d02-3524-4f28-934a-39135548b7a2)

**Reply by tommoor** [reply]
outline logs look good 👍 

https-portal is failing because there have been too many attempts to generate a certificate and lets encrypt limits quite aggressively.

If https-portal can't find outline then it's likely some sort docker networking issue

**Reply by dev30k** [reply]
Tried fixing it but there is this new error
![image](https://github.com/outline/outline/assets/66430325/023cc35b-9724-4dff-aaf3-05eaad6737ba)
